modcc/parser.cpp

```diff
@@ -973,6 +973,14 @@ symbol_ptr Parser::parse_function() {
     auto p = parse_prototype();
     if (p == nullptr) return nullptr;
 
+    // Functions may have a unit attached
```

Comment:
Is this related to the CMake update?

Reply:
Sorry I accidentally pushed the contents of another branch. I've removed it now.